lexer.cpp

```diff
@@ -1,15 +1,15 @@
-#line 1 "lexer.cpp"
+#line 2 "/Users/barret/_/git/R/libgraphqlparser/libgraphqlparser/lexer.cpp"
```

Comment:
this needs to be fixed up. I think cmake is running flex incorrectly; I just run it manually for commit.